analytics/terraform/spark-k8s-operator/helm-values/kube-prometheus-amp-enable.yaml

```diff
@@ -28,22 +28,11 @@ prometheus:
           resources:
             requests:
               storage: 50Gi
-    # Scrape Cost metrics for Kubecost and Yunikorn add-ons
+    # Scrape Cost metrics for Karpenter and Yunikorn add-ons
     additionalScrapeConfigs:
-      - job_name: kubecost
-        honor_labels: true
-        scrape_interval: 1m
-        scrape_timeout: 10s
-        metrics_path: /metrics
-        scheme: http
-        dns_sd_configs:
-          - names:
-              - kubecost-cost-analyzer.kubecost.svc
-            type: 'A'
-            port: 9003
```

Comment:
Are you not using KubeCost?

Reply:
Kubecost is still running but scraping these metrics was causing some problems with grafana dashboards. Queries were returning multiple metrics and breaking "group by" statements. 

Kubecost should still be able to collect metrics and make determinations on cost, it just isn't scraped in the central Prometheus config. 
happy to revert these changes as needed, i didn't investigate the issue very far and don't want to break other stuff.